macros/schema_tests/atleast_one.sql

```diff
@@ -0,0 +1,19 @@
+{% macro test_atleast_one(model, arg) %}
+
+select count(*)
+
+from
+
+  (
+    select *
+
+    from (
+       select
+        count({{ arg }}) count_of_rows
+
+        from {{ model }}
+    ) rows_with_values
+
+    where count_of_rows = 0 ) validation_errors
+
+{% endmacro %}
```

Comment:
@nave91 I think we can rewrite this as follows:

```sql
select
    count({{ arg }})
from {{ model }}
having count({{ arg }}) = 0
```

I _think_ these are equivalent, but lmk if you disagree!